### DISCUSSION TITLE
Additional stake showing as Delegation

### DISCUSSION THREAD

**Discussion by kw1knode** [subquery/subql#920] [Frontier TestNet - Bug Reporting]
![Screenshot 2022-04-26 155009](https://user-images.githubusercontent.com/25113494/165405758-c6cf8841-195d-4c5f-ad81-b724cb7681c4.png)

Upon adding additional SQT to my own stake, it now displays as `TOTAL DELEGATION TO INDEXER(S)` . Next era has not occurred but if displayed, this should fall under `TOTAL STAKE AMOUNT` . Being placed under delegating tab may cause some confusion.

**Comment by ianhe8x** [comment]
You are right, it should be consider your own stake. @CaiYiLiang

**Comment by lilian79511** [comment]
Ticket is created: SQN-411

**Comment by heyacherry** [comment]
Hello @kw1knode  Thanks for your issue reported! 
The issue fix has been released to production.
